main.tf

```diff
@@ -172,7 +172,7 @@ module "aurora_cluster" {
   rds_monitoring_interval = 30
 
   performance_insights_enabled          = var.performance_insights_enabled
-  performance_insights_kms_key_id       = var.performance_insights_enabled ? data.aws_kms_alias.rds.id : ""
+  performance_insights_kms_key_id       = var.performance_insights_enabled ? coalesce(var.performance_insights_kms_key_id, aws_kms_key.aurora_cluster_kms_key[0].arn) : ""
```

Comment:
I am not sure whether KMS key has to be created inside the module.

Reply:
Kms resource is written in inside the module already so if we remove then we can change it to default kms but currently its written 2 years ago

Reply:
is the create kms key being used any where?

Reply:
`aurora_cluster_kms_key` is not being used anywhere except i am using this in that performance attribute condition which can be removed if we are planning to remove the resource of aurora_cluster_kms_key and alias

@vijay-stephen  @jamescrowley321  @tsaucier-sf

Reply:
Performance insight should probably have its own as you've added.